students/ptfbanks/Lesson03/pf_mailroom.py

```diff
@@ -0,0 +1,93 @@
+#  MailRoom exercise lesson03 final task
+#    - Build, sort, print a table
+#--------------------------Global Region-----------------------------------#
+donations = [
+    ("Bob Williams", 5580.00),
+    ("Tom Haskell", 345.05),
+    ("Earl Jackson", 4553.45),
+    ("Les Thomas", 8934.00),
+    ("James Black", 154),
+    ("Bob Williams", 3245.50),
+    ("Tom Haskell", 3245.90),
+    ("Earl Jackson", 45.6),
+    ("Les Thomas", 325.00),
+    ("James Black", 1300),
+    ("Bob Williams", 1000),
+    ("Tom Haskell", 243.05),
+    ("Earl Jackson", 345.00),
+    ("Les Thomas", 3665.00),
+    ("James Black", 21)]
+
+#-----------------------------------Define Functions----------------------#
+#Prepare list of donors
+def get_names(list):
+    names = []
+    for entry in (list):
+        name = entry[0]
+        if name not in names:
+            names.append(name)
+    return(names)
+            
+def select_Key(table):
```

Comment:
remember to use snake_case -> `select_key`

a better input var name would be an `item` since you are getting here a single item from your list.